ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/HikariJdbcConnectionProvider.scala

```diff
@@ -89,7 +89,7 @@ class HikariJdbcConnectionProvider(
     }
   }
 
-  override def getStreamingConnection(): Connection =
+  override def getStreamingConnection: Connection =
```

Comment:
If this method is side-effecting, it's probably better to keep this (you can override a `def` without arguments with a `def`  with an empty tuple of arguments).